README.md

```diff
@@ -1,5 +1,7 @@
 # Config Validator
 
+[![OSS-Fuzz Status](https://oss-fuzz-build-logs.storage.googleapis.com/badges/config-validator.svg)](https://bugs.chromium.org/p/oss-fuzz/issues/list?sort=-opened&can=1&q=proj:config-validator)
```

Comment:
This doesn't exist: https://oss-fuzz-build-logs.storage.googleapis.com/badges/config-validator.svg

Reply:
It should within a day or two. The PR adding it to OSS-Fuzz was only submitted earlier today.